modules/migration-known-issues.adoc

```diff
@@ -31,3 +31,11 @@ ifdef::migrating-3-4[]
 
 * If you cannot install CAM 1.2 on an {product-title} 3 cluster, download the current `operator.yml` file, which fixes this problem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843059[*BZ#1843059*])
 endif::[]
+
+ifdef::migrating-4-2-4[]
+* In the current release (CAM 1.2), you cannot migrate a namespace that contains imagestreams from OpenShift Container Platform 4.4 or later. An error message similar to the following appears in the Velero log:
+
```

Comment:
Don't put blank lines between elements. That will seriously mess up your rendering. Remember that the target HTML format does not recognize blank lines. Also, if you add more bullet items to this list, they will not align correctly.

If a bullet item has elements that need to be aligned, you must put '+' on lines between them. In this case, do not put lines in. You can see from previous items in this list that bugs are inline, not separated.